tests/pytest/test_error_2006_0032.py

```diff
@@ -0,0 +1,117 @@
+""" Automation of Shadow Utils tests """
+
+from __future__ import print_function
+import subprocess
+import os
+import pytest
+
+
+class TestShadowUtilsErrors():
+    """
+    Automation of Shadow Utils tests
+    """
+    def test_error_2006_0032(self):
+        """
+        :title: Shadow-Utils: Tests if shadow-utils are
```

Comment:
It might be good to keep add the test prefix "IDM-SSSD-TC:" to the title.

Reply:
I've googled that but still haven't found a reference for what this is addressing.

Reply:
we are now not putting any prefix , so removing it